packages/renderer/src/lib/ui/CopyToClipboard.svelte

```diff
@@ -22,6 +22,6 @@ function copyTextToClipboard() {
     </button>
   </Tooltip>
 </div>
-<div class="mt-1 my-auto text-xs truncate {$$props.class || ''}" aria-label="{title} copy to clipboard" title="{title}">
+<div class="mt-1 my-auto truncate {$$props.class || ''}" aria-label="{title} copy to clipboard" title="{title}">
```

Comment:
You can modify line 15 to
```
<Tooltip tip="Copy to Clipboard" bottom class="{$$props.class || ''}">
```
so you can control `<span/>` that wraps the `<slot />`